SwiftUIStarter/SwiftUIStarter/NativeDisplayView.swift

```diff
@@ -0,0 +1,201 @@
+                        .font(.system(size: 16))
+                    }
+                    .padding(.horizontal, 16)
+                    
+                    // Display unit ID input and Get Unit button
+                    HStack(spacing: 12) {
+                        TextField("Display unit ID", text: $displayUnitID)
+                            .textFieldStyle(RoundedBorderTextFieldStyle())
+                        
+                        Button("Get Unit") {
+                            viewModel.getDisplayUnit(withID: displayUnitID)
+                        }
+                        .foregroundColor(.blue)
+                        .font(.system(size: 16))
+                    }
+                    .padding(.horizontal, 16)
+                    
+                    // Get all display units button
+                    Button("Get all display units") {
+                        viewModel.getAllDisplayUnits()
+                    }
+                    .foregroundColor(.blue)
+                    .font(.system(size: 16))
+                    .padding(.horizontal, 16)
+                    
+                    Divider()
+                        .padding(.top, 8)
+                }
+                .background(Color(.systemGroupedBackground))
+                
+                // Display units list
+                VStack(spacing: 0) {
+                    // Section header
+                    HStack {
+                        Text("ALL DISPLAY UNITS")
+                            .font(.caption)
+                            .fontWeight(.medium)
+                            .foregroundColor(.gray)
+                        Spacer()
+                    }
+                    .padding(.horizontal, 16)
+                    .padding(.top, 16)
+                    .padding(.bottom, 8)
+                    .background(Color(.systemGroupedBackground))
+                    
+                    // Display units list
+                    List {
+                        ForEach(viewModel.displayUnits.indices, id: \.self) { index in
+                            let displayUnit = viewModel.displayUnits[index]
+                            DisplayUnitRowView(
+                                displayUnit: displayUnit,
+                                onClick: {
+                                    viewModel.handleClick(displayUnit)
+                                },
+                                onView: {
+                                    viewModel.handleView(displayUnit)
+                                },
+                                onShowDetails: {
+                                    showAlert(title: "Display Unit Details", message: getDisplayUnitDetails(displayUnit))
+                                }
+                            )
+                        }
+                    }
+                    .listStyle(PlainListStyle())
+                }
+            }
+            .navigationTitle("Native Display")
```

Comment:
Those require iOS 15.0, you can use `.navigationBarTitle(_:displayMode:)` instead.